README.md

```diff
@@ -10,7 +10,79 @@ The `telements` library offers a set of customizable UI components written in `T
 
 This repository uses `yarn workspaces` and `typescript`
 
+### Running locally
 
+#### 1. Clone this repository to a local folder of your choice
+
+```bash
+cd path-to/folder-name
+git clone git@github.com:telekom/telements.git
+cd telements
+git checkout development
+```
+
+#### 2. Install dependencies
+
+This repository uses `yarn workspaces`, for the symlinks to work correctly please use `yarn` in favour of `npm`
+
+```bash
+yarn
+```
+
+#### 3. Build packages
+
+```bash
+# you'll need to build at least the components to be available for boilerplates
+yarn build:components
+
+# you'll need to run this if you want to deploy one or more boilerplates
+yarn build:boilerplates
+
+# run all above commands sequentially  
+yarn build
+```
+
+#### 4. Watch components for development
+
+> **NOTE:** Currently, every `components-{target}` needs to be watched independently, to do so please open a new `terminal` tab for each one you want to watch
+
+> **TODO:** Explore possibilities to watch all (selected) `components-{target}` packages with a `watch` command at once. E.g. `npm-run-all`
+
+```bash
+# Styles: All `components-{target}` packages depend on the `styles` package
+# Make sure you watch it, if you modify it's source files
```

Comment:
if you modify its source files? 

without `

Reply:
@tomhoefer fixed typo